src/server/components/lists/find/handlers/country.ts

```diff
@@ -0,0 +1,73 @@
+import { countriesList } from "server/services/metadata";
+import { validateCountryLower } from "server/models/listItem/providers/helpers";
+import type { NextFunction, Request, Response } from "express";
+import { getDbServiceTypeFromParameter } from "server/components/lists/searches/helpers/getDbServiceTypeFromParameter";
+import { getRedirectIfListIsEmpty } from "server/components/lists/find/helpers/getRedirectIfListIsEmpty";
+import { logger } from "server/services/logger";
+
+export async function redirectIfEmpty(req: Request, res: Response, next: NextFunction) {
+  if (req.session.answers?.country) {
+    next();
+    return;
+  }
+
+  const country = req.session.answers?.country ?? req.params.country;
+
+  const funeralDirectorsParamsToSkip = ["insurance", "repatriation"];
+  if (funeralDirectorsParamsToSkip.includes(country)) {
```

Comment:
Hang on. So on line **9**. If `req.session.answers?.country` is true then the code won't get to line **14** so should like **14** just be:
```js
const country = req.params.country;
```

Reply:
Also, sorry just getting my head around this. Will the condition on line **17** ever be true?

Because the country string will never exist in the `funeralDirectorsParamsToSkip` array. Or maybe I'm missing something 🤔